erpnext/accounts/doctype/sales_taxes_and_charges/sales_taxes_and_charges.json

```diff
@@ -212,13 +214,30 @@
    "label": "Account Currency",
    "options": "Currency",
    "read_only": 1
+  },
+  {
+   "columns": 2,
+   "fieldname": "net_amount",
+   "fieldtype": "Currency",
+   "in_list_view": 1,
+   "label": "Net Amount",
+   "options": "currency"
+  },
```

Comment:
@blaggacao the `net_amount` fields should probably be read-only as well, right?

Reply:
https://github.com/frappe/erpnext/pull/44215